### DISCUSSION TITLE
PostgreSQL table create/update text/varchar data type issue

### DISCUSSION THREAD

**Discussion by hardc0l2e** [Budibase/budibase#2238] [Help] [answered]
When creating/updating a record/row on a PostreSQL database table, if the text start with a number, saving it will automatically remove the rest of content, just leaving the value with number only.

For example the value should be "**123 XYZ Company**" it will only save "**123**"

![20210804_01 13 48 AM-Budibase_-_Google_Chrome](https://user-images.githubusercontent.com/1472421/128058098-9fe47b05-e066-4f84-9c0b-05f9b15b40c0.png)
![20210804_01 14 06 AM-Budibase_-_Google_Chrome](https://user-images.githubusercontent.com/1472421/128058112-01738251-2c85-4c56-bffe-d3ff78c0125a.png)

**Comment by hardc0l2e** [comment]
Here's a demo of the problem, but when you directly create or update tables from PostgreSQL CLI or tools like PgAdmin, there's no problem, it accepts the value without removing the content after the number. It only happens when creating/updating rows via budibase.

![Jn9vc6AXAW](https://user-images.githubusercontent.com/1472421/128099567-99c96e0f-9a8e-45ec-bc14-f396d70ed1ec.gif)

**Comment by mjashanks** [comment, marked as answer]
Thanks for this @hardc0l2e - I can reproduce this also.

I have logged this as a bug here: https://github.com/Budibase/budibase/issues/2250.

**Reply by hardc0l2e** [reply]
Thanks.
